extensions/ql-vscode/package.json

```diff
@@ -346,6 +346,12 @@
           "default": "",
           "patternErrorMessage": "Please enter a valid folder",
           "markdownDescription": "The name of the folder where we want to create queries and query packs via the \"CodeQL: Create Query\" command. The folder should exist."
+        },
+        "codeQL.autogenerateQlPacks": {
```

Comment:
In some places (e.g. the `codeQL.createQuery.folder` setting description) we're using the term "query pack" and in some places "QL pack". Are they the same, or if so can we standardise? If not what's the difference and does this need to be communicated?

Reply:
They are the same, yup. The default way to call them is `QL pack` or `CodeQL pack`. However the command to create it is called `Create Query` because we don't want to user to know the details of how we create the query. This was an intentional design/product decision.

Reply:
I could rename both settings to be something like:
```
codeQL.createQuery.folder -> codeQL.createQuery.qlPackLocation
codeQL.autogenerateQlPacks -> codeQL.createQuery.autogenerateQLPacks
```

Reply:
Those setting names look good to be. The name "create query" is fine. I think it makes sense that the command is called that but then some of the settings for controlling it also contain references to "QL packs". But basically the name "query pack" should never appear anywhere.

Reply:
Makes sense. There are quite a few places where we use "query pack" and I'm a bit weary of renaming them in this PR. I've fixed all the cases related to this flow, so hopefully it keeps it consistent.